### DISCUSSION TITLE
Home Lab - Inexpensive Hardware Suggestions

### DISCUSSION THREAD

**Discussion by secraven** [Security-Onion-Solutions/securityonion#12259] [General]
I am trying to find some used inexpensive hardware for my home lab.  I am looking for suggestions that are hopefully relatively efficient and doesn't create too much heat.  I've been looking at used Lenovo Thinkcentre systems.  I don't need long term traffic retention for my lab setup.  A few days of analysis would be fine for my lab.  Sorry if this is posted elsewhere, but I couldn't find any recent threads.

**Comment by reyesj2** [comment]
Inexpensive is subjective. Personally, I use an old gaming desktop to run proxmox for VMs. I have been recommended a Dell Precision T7810 Workstation for this purpose as well. Older tower system you can find for around $400

**Reply by secraven** [reply]
Thanks! $400 or less would be great.  I had a Dell Tower previously but its was quite large and overkill.  ESXi is no longer a free option unfortunately.   An all in one systems with up to about a week of logs would be great.

**Reply by InfosecGoon** [reply]
I'm using that same Dell workstation for my Proxmox and it's been great - enough horsepower for home network infrastructure stuff (Security Onion, TrueNAS, Pi-hole) with plenty left over to spin up VMs to play with new tools.

**Reply by oneCrazyAdmin** [reply]
What do you use to get the data into Security Onion?  Span port on a switch?

**Reply by secraven** [reply]
I have a network TAP - Dualcomm ETAP-2003 10/100/1000.

**Reply by oneCrazyAdmin** [reply]
Thanks!  I will have to take a look
